src/rules/at-rule-empty-line-before/index.js

```diff
@@ -6,6 +8,7 @@ import {
   ruleMessages,
   validateOptions,
 } from "../../utils"
+import { cssStatementIsIgnoredAtRule } from "../block-opening-brace-space-before"
```

Comment:
Yes, we don't want these rules depending on each other. Would be better with a util.